packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx

```diff
@@ -151,8 +151,8 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
                 )}
 
                 {user.data?.ability.can(
-                    'update',
-                    subject('Project', { organizationUuid, projectUuid }),
+                    'manage',
+                    subject('PinnedItems', { organizationUuid, projectUuid }),
```

Comment:
We need the same fix on packages/frontend/src/components/Explorer/SpaceBrowser/SpaceBrowserMenu.tsx